syntax = "proto3";


message Product {
    int32 id  = 1;
    int64 created_at = 2; 
    int64 updated_at = 3;
    string name = 4;
    string description = 5;
    string category = 6;
    float cost_price = 7;
    float sale_price = 8;
    float discount = 9;
}
